app/src/androidTest/java/ch/epfl/sdp/blindly/fake_module/FakeUserCacheModule.kt

```diff
@@ -19,18 +19,22 @@ import javax.inject.Singleton
 open class FakeUserCacheModule {
     companion object {
         private const val username = "Jane Doe"
-        private val location = createLocationTableEPFL()
+        private val location = createLocationTableEPFL() // Ecublens, Switzerland
+        private val location2 = listOf(47.749, 7.335) // Mulhouse, France

```

Comment:
I think it would be clearer if the numbers are named `MULHOUSE_LAT` and `MULHOUSE_LNG`.